src/index.html

```diff
@@ -17,11 +17,434 @@
+              >
+                Apple
+              </a>
+            </li>
+            <li class="navigation__container-list">
+              <a
+                href="#"
+                class="navigation__container-list-link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="navigation__container-list">
+              <a
+                href="#"
+                class="navigation__container-list-link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="navigation__container-list">
+              <a
+                href="#"
+                class="navigation__container-list-link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="navigation__container-list">
+              <a
+                href="#"
+                class="navigation__container-list-link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="navigation__container-list">
+              <a
+                href="#"
+                class="navigation__container-list-link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="navigation__container-list">
+              <a
+                href="#"
+                class="navigation__container-list-link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="navigation__container-list">
+              <a
+                href="#"
+                class="navigation__container-list-link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="cards">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__product">
+          <div class="card__product-image">
+            <img
+              class="card__product-image"
+              src="./images/imac.jpeg"
+              alt="imac-image"
+            />
+          </div>
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__product-code">Product code: 195434</p>
+        </div>
+        <div class="card__info">
+          <div class="card__info-reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__info-reviews-amount">Reviews: 5</p>
+          </div>
+
+          <div class="card__info-price">
+            <div class="card__info-price-tag">Price:</div>
```

Comment:
Use the semantic tag for text everywhere
```suggestion
            <p class="card__info-price-tag">Price:</p>
```